assets/app.js

```diff
@@ -1611,6 +1643,96 @@ new Vue({
             })
             return false
 
+        },
+        resolveRejectedProducts: function () {
+            this.products = [];
+            console.log("test", this.rawRejectedProducts)
+            products = this.rawRejectedProducts
+            //action for getting products
+            var product_names = []
+            for (var key in products) {
+                if (key !== 'success' && key !== 'metatags' && key !== 'materials' && 'propertyFormula' in products[key]) {
```

Comment:
nit(optional): not a biggy, but for DRY, I'd do something like `['success', 'metatags', 'materials'].indexOf(key) === -1`, actually I'd define a separate constant so in future if key can't be say, 'attributes' or something, I could just add an entry in array :slightly_smiling_face: , and this line could be shorter.

Reply:
I agree